src/main/java/de/dagere/peass/ci/logs/rts/AggregatedRTSResult.java

```diff
@@ -15,9 +15,10 @@ public AggregatedRTSResult(final RTSLogSummary logSummary, final RTSResult resul
       this.logSummary = logSummary;
       this.result = result;
       if (logSummary != null) {
-         rtsAnyError = logSummary.isErrorInCurrentVersionOccured() || logSummary.isErrorInPredecessorVersionOccured();
-         rtsAllError = logSummary.isErrorInCurrentVersionOccured() && (!logSummary.isVersionContainsSuccess()
-               || !logSummary.isPredecessorContainsSuccess());
+
+         rtsAnyError = logSummary.isErrorInCurrentVersionOccured() || logSummary.isErrorInPredecessorVersionOccured() && !(logSummary.isPredecessorContainsParametrizedwhithoutIndex() || logSummary.isVersionContainsParametrizedwhithoutIndex());
+         rtsAllError = logSummary.isErrorInCurrentVersionOccured() && (!logSummary.isVersionContainsSuccess() || !logSummary.isPredecessorContainsSuccess()) && !(logSummary.isPredecessorContainsParametrizedwhithoutIndex() || logSummary.isVersionContainsParametrizedwhithoutIndex());
```

Comment:
It would be nicer to group these into local booleans, to make it understandable to future developers (as a rule of thumb, more then three booleans compared on one line would be too much).

Reply:
it is fixed